cryptus-app/components/wallet_viewer/nft_details/nft_infos/nft_buy_button/nft_buy_button.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "/styles/variables" as v;
+
+.container {
+    align-self: flex-start;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    width: fit-content;
```

Comment:
Put a min-width, so most of the time the button looks the same (like for 1, 10 or 100 eth), and only if it is larger, then width will fit content.